libs/barista-components/filter-field/src/filter-field.ts

```diff
@@ -827,6 +841,223 @@ export class DtFilterField<T = any>
     }
   }
 
+  /** @internal Handles keyup on the input */
+  _handleInputKeyUp(event: KeyboardEvent): void {
+    if (!this._tags.length) {
+      return;
+    }
+
+    this._getCurrentFocusState();
+
+    const keyCode = _readKeyCode(event);
+    if (keyCode === LEFT_ARROW && this._currentFocusState === 'inputboundry') {
```

Comment:
You only react to things, if `currentFocusState` is `inputboundary`, maybe we can add a check first, and early exit.

```
if (this._currentFocusState !== 'inputboundary') {
  return;
}
```

Reply:
This seems to be still in the code. I'm fine with it, consider it a nitpick.